hoomd/hpmc/external/field.py

```diff
@@ -329,8 +309,8 @@ def add_field(self, fields):
             self.cpp_compute.addExternal(field.cpp_compute)
 
 
-class wall(_external):
-    R""" Manage walls (an external field type).
+class wall(_External):  # noqa: name will change in v3
```

Comment:
`wall` doesn't belong in the `field.py` file. You can create a `wall.py` to keep the old v2 code for now and add it to the exceptions for pre-commit and flake8 instead of specifying noqa on every line.